src/actions/actions.js

```diff
@@ -268,11 +269,7 @@ export function loginFailure() {
 }
 
 export function login(successCallback, failCallback) {
-  const url = '/oauth/login/github'
-  const name = 'github_login'
-  const specs = 'width=500,height=600'
-  const authWindow = window.open(url, name, specs)
-  authWindow.focus()
+  loginToServer()
   return (dispatch) => {
     // Functions to be called by child window
```

Comment:
Modify the comment here to make it more clear that the child window is created by loginToServer.

```js
// Functions to be called by child window set up by loginToServer()
```